vowels-component.jsx

```diff
@@ -0,0 +1,100 @@
+import * as React from "react";
+import "./styles.css";
+
+export default function App() {
+  const [name, setName] = React.useState("");
+  const [records, setRecords] = React.useState([]);
+  const vowelsfreq = (name) => {
```

Comment:
On every re-render, the `vowelsfreq` and `highlightVowels` functions will be re-created.
They do not use anything from the `App` component. We can move these 2 functions outside of the functions.